arm-timeseriesinsights/2017-02-28-preview/Swagger/timeseriesinsights.json

```diff
@@ -0,0 +1,1780 @@
+            "schema": {
+              "$ref": "#/definitions/EnvironmentResource"
+            }
+          },
+          "404": {
+            "description": "The subscription, resource group, or environment could not be found."
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Environments"
+        ],
+        "operationId": "Environments_Delete",
+        "x-ms-examples": {
+          "EnvironmentsDelete": { "$ref": "../examples/EnvironmentsDelete.json" }
+        },
+        "description": "Deletes the environment with the specified name in the specified subscription and resource group.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/EnvironmentNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The environment was successfully deleted."
+          },
+          "204": {
+            "description": "The environment was successfully deleted."
+          },
+          "404": {
+            "description": "The subscription, resource group, or environment could not be found."
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.TimeSeriesInsights/environments": {
+      "get": {
+        "tags": [
+          "Environments"
+        ],
+        "operationId": "Environments_ListByResourceGroup",
+        "x-ms-examples": {
+          "EnvironmentsByResourceGroup": { "$ref": "../examples/EnvironmentsListByResourceGroup.json" }
+        },
+        "description": "Lists all the available environments associated with the subscription and within the specified resource group.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Environments returned successfully.",
+            "schema": {
+              "$ref": "#/definitions/EnvironmentListResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.TimeSeriesInsights/environments": {
+      "get": {
+        "tags": [
+          "Environments"
+        ],
+        "operationId": "Environments_ListBySubscription",
```

Comment:
Is this a pageable operation?

Refer: https://github.com/Azure/autorest/tree/master/docs/extensions#x-ms-pageable